samples/geolocation-weather/src/index.ts

```diff
@@ -0,0 +1,99 @@
+// For AutoRouter documentation refer to https://itty.dev/itty-router/routers/autorouter
+import { AutoRouter } from 'itty-router';
+import { getClientAddressFromRequest, cleanupIpAddress } from "./helpers";
+import { Variables } from "@fermyon/spin-sdk";
+
+
+let router = AutoRouter();
+
+// Route ordering matters, the first route that matches will be used
+// Any route that does not return will be treated as a middleware
+// Any unmatched route will return a 404
+router.get("/", (request, {token}) => getWeather(request, token)) 
+
+//@ts-ignore
+addEventListener('fetch', async (event: FetchEvent) => {
+   let token = Variables.get("waqi_api_token");
+   if (!token) {
+     event.respondWith(new Response("Internal Server Error", {status: 500}));
```

Comment:
It would also possibly make sense to leave a console.log here as to why the request failed to make it easier to debug. My apologies for not leaving it with the previous review.

Reply:
updated 👍